testsuite/features/secondary/min_salt_formulas.feature

```diff
@@ -59,7 +59,7 @@ Feature: Use salt formulas
      When I click on the "disabled" toggler
      And I click on "Apply Highstate"
      Then I should see a "Applying the highstate has been scheduled." text
-     And I wait until event "Apply highstate in test-mode scheduled by admin" is completed
+     And I wait at most 300 seconds until event "Apply highstate in test-mode scheduled" is completed
```

Comment:
is the `by admin` part no longer necessary or was it removed accidentally?

Reply:
Actually I did it in purpose, we don't need to always check that the even is triggered by the only user we have :)
I think we can just short steps without it, in cases like this one.